### PR TITLE
Testing failure on travis

### PR DESCRIPTION
**Pull request by vijetmahabaleshwar-okta**

### CODE REVIEW

File: scripts/runTests.sh [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
+    if [ $? -ne 0 ]; then
+        STATUS=1
+        FAILED_PACKAGE=$package
+    fi
     cd ../..
 done
+
+if [ ! -z $FAILED_PACKAGE ]; then
+    echo $FAILED_PACKAGE
+fi
```
Comment (jmelberg-okta) [inline scripts/runTests.sh]: Instead of relying on the var `STATUS`, we can exit inside of this conditional once we `echo $FAILED_PACKAGE`

Comment (robertjd) [reply scripts/runTests.sh]: Multiple packages could fail.  The goal of this fix should be to exit non-zero if any of them fail.  The individual tests write error information to output so we already know which ones failed.

Comment (jmelberg-okta) [reply scripts/runTests.sh]: Makes sense. Just to clarify - are you saying we should have a hard fail once **any** package fails?

For example, if `okta-angular` fails - exit immediately regardless of the status of other tests.

Comment (robertjd) [reply scripts/runTests.sh]: We should probably exit immediately, since the total test run time is already getting close to 10 minutes

Comment (vijetmahabaleshwar-okta) [reply scripts/runTests.sh]: Updated the script to exit immediately on failure.

**Pull request merged**